DOC-4181 (#716)
corda/corda-docs-portal: content/en/platform/corda/5.0-dev-preview-2/getting-started/get-started.md

> Developer Preview 1 was effectively a taster for the Corda 5 API. It was however, mostly based on Corda 4 under the hood. Developer Preview 2 is the real deal, based entirely on the new Corda 5 code base with possibly material differences in the API. As a result, code written for Developer Preview 1 is unlikely to work with Developer Preview 2 without modification. You can not use any artefacts released from Developer Preview 1 for Developer Preview 2.  

Reviewer: ```suggestion
Developer Preview 1 was effectively a taster for the Corda 5 API. It was, however, mostly based on Corda 4 under the hood. Developer Preview 2 is the real deal, based entirely on the new Corda 5 code base with possibly material differences in the API. As a result, code written for Developer Preview 1 is unlikely to work with Developer Preview 2 without modification. You cannot use any artefacts released from Developer Preview 1 for Developer Preview 2.  
```
